ChatQnA/README.md

```diff
@@ -16,6 +16,8 @@ Quick Start Deployment Steps:
 2. Run Docker Compose.
 3. Consume the ChatQnA Service.
 
+Note: If you do not have docker installed you can run this script to install docker : bash docker_compose/install_docker.sh
```

Comment:
Note: If you do not have docker installed you can run this script to install docker `bash docker_compose/install_docker.sh`.

Reply:
done